skimage/_shared/utils.py

```diff
@@ -113,6 +114,61 @@ def fixed_func(*args, **kwargs):
         return fixed_func
 
 
+def docstring_add_deprecated(func, kwarg_mapping, deprecated_version=None):
+    """Add deprecated kwarg(s) to the "Other Params" section of a docstring.
+
+    Parmeters
+    ---------
+    func : function
+        The function whos docstring we wish to update.
+    kwarg_mapping : dict
+        A dict containing {old_arg: new_arg} key/value pairs as used by
+        `deprecate_kwarg`.
+    deprecated_version : str, optional
+        A major.minor version string specifying when old_arg was
+        deprecated.
+
+    Returns
+    -------
+    new_doc : str
+        The updated docstring. Returns the original docstring if numpydoc is
+        not available.
+    """
+    try:
+        from numpydoc.docscrape import FunctionDoc, Parameter
+    except ImportError:
+        # Return an unmodified docstring if numpydoc is not available.
+        return func.__doc__
+
+    Doc = FunctionDoc(func)
+    for old_arg, new_arg in kwarg_mapping.items():
+        if deprecated_version is None:
```

Comment:
Why do we want to handle this in scikit-image? Don't we want to encourage contributors and core developers to fill this information out?

Reply:
yeah, we could just require it. For the handful of cases that weren't just added in 0.19 I would have to check when they were added, but that shouldn't take too long.

Reply:
Thanks!